compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -378,21 +378,22 @@ execCompile inputFile outputFile opts =
           setFilesOfInterest ide (Set.singleton inputFile)
           runAction ide $ do
             when (optWriteInterface opts') $ do
+
+                -- Support for '-ddump-parsed', '-ddump-parsed-ast', '-dsource-stats'.
```

Comment:
Why is this conditional on writing interface files?

Reply:
Good catch, this should move outside and before the `when`.

Reply:
On it.